src/simulation/simulators/olbMixing.hh

```diff
@@ -395,11 +395,21 @@ void lbmMixingSimulator<T>::storeConcentrations(std::unordered_map<int, std::uno
     this->concentrations = concentrations_;
 }
 
+template<typename T>
+void lbmMixingSimulator<T>::storeNodeConcentrationFields(std::unordered_map<int, std::unordered_map<int, std::vector<T>>> concentrationFieldsOut_) {
```

Comment:
Here, the logic from the simulator side is still missing (i.e. the concentrationFieldsOut are not used in the subsequent code)

Reply:
This means that here would be the "coupling point" for the Abstract -> CFD?
Could you please add a TODO comment here for me, as reference point? E.g.,
```
/**
 * TODO: coupling Abstract -> CFD
 */
```